src/components/CollapsibleList/CollapsibleListEntry/CollapsibleListEntry.tsx

```diff
@@ -0,0 +1,78 @@
+import { ExpandMore } from '@common/icons'
+import { memo } from 'react'
+import {
+  AccordionDetails,
+  AccordionSummary,
+  Typography,
+  Accordion,
+  ListItem,
+  Divider,
+  List,
+  Zoom
+} from '@common/components'
+
+/**
+ * @prop body - The body text of an entry, displayed as a bullet points.
+ * @prop header - The header text of an entry.
+ * @interface
+ */
+export type CollapsibleListEntryContentProps = {
+  body?: string[]
+  header?: string
+}
+
+/**
+ * @prop startAnimation - Whether the animation should start or not.
+ * @prop delay - The delay of the animation in ms.
+ * @prop {@link CollapsibleListEntryContentProps} - The content of a single CollapsibleListEntry.
+ * @interface
+ */
+type CollapsibleListEntryProps = CollapsibleListEntryContentProps & {
```

Comment:
lol sowas habe ich noch nie gesehen, Interessant

Reply:
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces